panel/widgets/slider.py

```diff
@@ -25,6 +25,8 @@
 from bokeh.models.widgets.sliders import NumericalSlider as _BkNumericalSlider
 from param.parameterized import resolve_value
 
+from panel.models.datetime_slider import DatetimeSlider as _PnDatetimeSlider
```

Comment:
Use relative import here. 

The convention also seems to still use `_Bk` as a prefix

https://github.com/holoviz/panel/blob/c65c390c7357e1a8c3c3346feadef33a3c41eba0/panel/widgets/select.py#L30

Reply:
You forgot to change it to an relative import.

Reply:
Ah yeah I forgot. Updated it now.